pkg/systemd/services/service-details.jsx

```diff
@@ -105,8 +104,14 @@ ServiceConfirmDialog.propTypes = {
  *      Method for calling unit methods like `UnitStart`
  *  - fileActionCallback
  *      Method for calling unit file methods like `EnableUnitFiles`
+ *  - deleteActionCallback
+ *      Method for calling deleting the systemd timer
  *  - disabled
  *      Button is disabled
+ *  - isCustom
+ *      Unit is installed in /etc/systemd/system
+ *  - isTimer
+ *      Unit is a timer
```

Comment:
Proposal for generalization: Leave the policy decision to the caller, and offer the "Delete" action iff `deleteActionCallback` is defined. We may want to delete other units in the future, and this avoids some awkward inconsistencies wrt. "this gets passed as property" vs. "the deleteAction() code implicitly assumes a timer anyway".